exercises/practice/resistor-color/.meta/example.gd

```diff
@@ -0,0 +1,17 @@
+const COLORS = ["black",
+                "brown",
+                "red",
+                "orange",
+                "yellow",
+                "green",
+                "blue",
+                "violet",
+                "grey",
+                "white"]
+
```

Comment:
@BNAndras I would also put 2 blank lines between a const and the first method ;)